packages/console/src/pages/ConnectorDetails/components/SenderTester/index.tsx

```diff
@@ -9,7 +9,7 @@ import Button from '@/components/Button';
 import FormField from '@/components/FormField';
 import TextInput from '@/components/TextInput';
 import Tooltip from '@/components/Tooltip';
-import { phoneRegEx, emailRegEx } from '@/utils/regex';
```

Comment:
Prefer `utils`? idk... but `utils` is more like a convention, similar to `config` instead of `configuration`, `i18n` instead of `internationalization`
Thoughts?

Reply:
utilities it's ok i think (less than 10 chars)